src/Core_Language_Command.php

```diff
@@ -316,7 +316,7 @@ public function update( $args, $assoc_args ) { // phpcs:ignore Generic.CodeAnaly
 	}
 
 	/**
-	 * Activates a given language.
+	 * Activates a given language. (Deprecated: use wp site switch-language instead)
```

Comment:
Can we put this on a new line, and make it similar to the deprecation notice for `wp scaffold block`? https://github.com/wp-cli/scaffold-command/blob/594f0b47a2af6a19664f0f4717998b2cf5cb3ed7/src/Scaffold_Command.php#L205-L208

Reply:
Sure! On it 👍.

Reply:
Hello @danielbachhuber,
Have updated the doc-comment as suggested.

Please do let me know if further changes are required 😃.

cc: @swissspidy

Reply:
@Pathan-Amaankhan Thanks! I made one more minor tweak eeb97ffade42b002ca6bb9b756d7731f8d33846d